includes/misc-functions.php

```diff
@@ -58,6 +58,13 @@ function edd_get_admin_url( $args = array() ) {
  */
 function edd_is_test_mode() {
 	$ret = edd_get_option( 'test_mode', false );
+
+	// Override any setting with the constant.
+	if ( edd_is_test_mode_forced() ) {
+		$ret = true;
+	}
+
+	// At the end of the day, the filter still has the final say.
```

Comment:
I kind of don't love allowing the filter to override the constant, but I see it's consistent with the `EDD_DEBUG_MODE` constant behavior. (Note: it would be neat if that constant got the same treatment as this.)

Reply:
@robincornett Yeah I thought about that too, but my thought process was...

Since adding a filter requires some development work if it's in there already (prior to this release) it is in there for a reason, and since adding a filter is harder than a constant, it will prevent someone from overriding a deliberate decision by a developer to have test mode set through the filter.